src/modules/users/users.service.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Injectable,
+  ConflictException,
+  NotFoundException,
+} from '@nestjs/common';
+import { CreateUserDto, UpdateUserDto } from './dto/user.dto';
+import { DatabaseService } from '../database/database.service';
+
+@Injectable()
+export class UsersService {
+  constructor(private database: DatabaseService) {}
```

Comment:
we usually want to keep 
`private database: DatabaseService`
 to be
`private databaseService: DatabaseService`

as we are not working directly with the database, but with the service (Same for future implementations)